src/gui/sdl_mapper.cpp

```diff
@@ -111,6 +111,9 @@ static CBindList holdlist;
 
 static bool suppress_save_mapper_file_message = false;
 
+// In SDL ticks (msec)
+static Uint32 joystick_subsystem_init_timestamp = std::numeric_limits<uint32_t>::max();
```

Comment:
Our standard uint32_t should probably work, too.

Reply:
Yeah, or just `auto`.

Reply:
[changed](https://github.com/dosbox-staging/dosbox-staging/compare/7f7da33da32a1bb1aef484278a7b68e2dda667de..9fcf31a7a33415fd028d9539cd06f3f5d453650e) to `auto`.